app/controllers/candidate_interface/restructured_work_history/base_controller.rb

```diff
@@ -0,0 +1,17 @@
+module CandidateInterface
+  class RestructuredWorkHistory::BaseController < CandidateInterfaceController
+    before_action :redirect_to_dashboard_if_submitted, :render_404_if_candidate_has_used_existing_flow_or_flag_is_active
+
+    def show; end
```

Comment:
In the interest of establishing conventions, I think Base should be for common behaviours, and something like StartController for the first step. Clearer separation of concerns.

Reply:
I agree. That's a good point. I will update now.